deploy/single/all-in-one-postgres.yaml

```diff
@@ -706,13 +712,13 @@ spec:
         command: [ "/bin/sh", "-c", "until nc -zv $KONG_PG_HOST $KONG_PG_PORT -w1; do echo 'waiting for db'; sleep 1; done" ]
       containers:
       - name: kong-migrations
-        image: kong:0.14.1-centos
+        image: kong:1.0.0-centos
         env:
         - name: KONG_PG_PASSWORD
           value: kong
         - name: KONG_PG_HOST
           value: postgres
         - name: KONG_PG_PORT
           value: "5432"
-        command: [ "/bin/sh", "-c", "kong migrations up" ]
```

Comment:
I'm still learning the new migration framework, correct me if I understand it not right:
1) fresh install of 1.0: migrations bootstrap
2) migration from 0.14 to 1.0: migrations up
3) migrations between version after 1.0: migrations up

to cover all use cases, we might want to use something like `kong migrations up || kong migrations bootstrap` ?